app/controllers/api/v1/users_controller.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   module V1
     class UsersController < Api::V1::ApiController
+      before_action :auth_user
```

Comment:
I think you need to do this same thing in others controllers:
`passwords_controller.rb      registrations_controller.rb  sessions_controller.rb       settings_controller.rb`

Reply:
I though about it but those only add callbacks and extends `DeviseTokenAuth` what would you auth there?

Reply:
oh you are right